### DISCUSSION TITLE
demo.vercel.store with exceptionally low Lighthouse score

### DISCUSSION THREAD

**Discussion by devklepacki** [vercel/next.js#18473] [Help]
For me Lighthouse score was as low as 55. This propaply shouldn't happen as on the Next.js Conf analyzing and performance were talked about so much. I don't know, just dropping it here. Of course these are very early days since Next 10 release, but still, how could this be pushed to us in times when we all aim for <1s scores?

![image](https://user-images.githubusercontent.com/20723654/97578148-aff75700-19f0-11eb-802f-9a59c29622a3.png)

**Comment by devklepacki** [comment]
With Google's PageSpeed Insights it's even lower with the score of 48:
https://developers.google.com/speed/pagespeed/insights/?url=demo.vercel.store

> Origin Summary: Over the previous 28-day collection period, the aggregate experience of all pages served from this origin **does not pass** the Core Web Vitals assessment.

**Comment by adamsvystun** [comment]
I guess it should be compared with what it is possible to get with that kind of website, but still, I expected it to have ~90 scores after watching the presentation.

**Comment by adamsvystun** [comment]
Okay, figured it out. You want to look at averages across users. Page insights gives a distribution:

![image](https://user-images.githubusercontent.com/15347211/103108385-c39bf400-4614-11eb-993f-cb569e0f7d02.png)

As you can see the majority of users have 90+ scores.

**Reply by devklepacki** [reply]
@adamsvystun it could change in a meantime of course. At the time of my writing the message was clear: _the aggregate experience of all pages served from this origin **does not pass** the Core Web Vitals assessment._ Anyway, that's great it does right now.
